mesos_master/tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import copy
+import pytest
+
+from datadog_checks.dev import docker_run
+
+from . import common
+
+
+@pytest.fixture(scope='session')
+def dd_environment(instance):
+    compose_file = os.path.join(common.HERE, 'compose', 'docker-compose.yml')
+
+    with docker_run(compose_file):
+        yield instance
+
+
+@pytest.fixture(scope='session')
```

Comment:
I believe setting the scope to `session` beats the purpose of copying the instance, since it will only actually call the `instance` function once in the session, and then keep a reference to the returned value. (not 100% sure, but I that's what would make sense IMO for the `session` scope)
Since setting this scope is necessary to use the fixture in `dd_environment`, let's just yield the constant `common.INSTANCE` directly, and remove the scope here.